verticapy/_config/config.py

```diff
@@ -148,6 +153,182 @@ def set_option(key: str, value: Any = None) -> None:
         using iterative functions.
     value: object, optional
         New value of the option.
+
+    Examples
+    --------
+    .. code-block:: python
+
+        from verticapy.datasets import load_titanic
```

Comment:
Please separate the import of the dataset from the set_option.
1. First block should be the import of the dataset: Let's import the titanic dataset for this example.
.. hint:: VerticaPy provides multiple datasets accessible through loaders from the dataset module.
2. Second block should be the import of the `set_option`. You should explain each option with one simple line. Please use direct text rather than comment whenever it is possible.